plugins/cpp_metrics/service/cxxmetrics.thrift

```diff
@@ -4,24 +4,61 @@ include "project/project.thrift"
 namespace cpp cc.service.cppmetrics
 namespace java cc.service.cppmetrics
 
-enum CppMetricsType
+enum CppUnitType
```

Comment:
This is not used anywhere. Should be removed?

Reply:
Yes.